patches/html-to-image+1.8.5.patch

```diff
@@ -0,0 +1,41 @@
+diff --git a/node_modules/html-to-image/.DS_Store b/node_modules/html-to-image/.DS_Store
```

Comment:
Is there an upstream PR for this patch so that we don't have to maintain it forever?